folly/lang/ToAscii.h

```diff
@@ -59,7 +59,7 @@ namespace detail {
 FOLLY_ERASE auto to_ascii_port_clzll(uint64_t v) {
 #if _MSC_VER
 #if FOLLY_X64
-  return __lzcnt64(v);
+  return static_cast<int>(__lzcnt64(v));
```

Comment:
Let's move this cast into the caller, since the return type of this function is intended to be the return type of the intrinsic or the builtin which it calls.

Reply:
ahh yes, I didn't notice the return type had changed to auto. Thanks!

Reply:
I tested it locally and looks like the 'auto' keyword takes care of the warning for us, so I will just remove this change altogether.

Reply:
@yfeldblum we're getting some undefined behavior with x86 using auto return type here, this method returns void so casting in the caller is not enough. Any insight for the purpose of line 63-64? Is this expected to never be called from x86?

Reply:
The type of the expression `__assume(0), 0` should be `int`.